app/events/[id]/page.tsx

```diff
@@ -0,0 +1,52 @@
+'use client'
+
+import LandingLayoutView from '@hhs/layouts/landing-layout';
+import Image from 'next/image';
+import { useParams } from 'next/navigation'
+import { useEffect, useState } from 'react';
+
+
+const EventDetailPage = () => {
+    const { id } = useParams();
+    const [event, setEvent] = useState(null);
+
+    const getEvent = (id) => {
```

Comment:
Service operations related to 'kommunity' need to be moved to the 'lib' folder @atiilla